environment.yml

```diff
@@ -9,10 +9,11 @@ channels:
 dependencies:
   - python=3.9
   - pytorch-cuda=11.8
+  - lightning=2.0.1
```

Comment:
Maybe comment on why we're pinning this specific version -- we want to be as permissive as possible with our dependencies, but if there's a reason why we can't use >= 2.0.1, we should at a minimum document it for future reference so we can loosen this restriction when possible.